[2021-12-22 17:37:10,814] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:27:00+00:00 [queued]>
[2021-12-22 17:37:10,818] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:27:00+00:00 [queued]>
[2021-12-22 17:37:10,818] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:37:10,818] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:37:10,818] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:37:10,828] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): face_crop> on 2021-12-20 00:27:00+00:00
[2021-12-22 17:37:10,831] {standard_task_runner.py:52} INFO - Started process 29832 to run task
[2021-12-22 17:37:10,835] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'face_crop', 'scheduled__2021-12-20T00:27:00+00:00', '--job-id', '156', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmps6gwf4_x', '--error-file', '/tmp/tmpp9m5038i']
[2021-12-22 17:37:10,835] {standard_task_runner.py:77} INFO - Job 156: Subtask face_crop
[2021-12-22 17:37:10,859] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.face_crop scheduled__2021-12-20T00:27:00+00:00 [running]> on host myamya
[2021-12-22 17:37:10,891] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=face_crop
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:27:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:27:00+00:00
[2021-12-22 17:37:10,893] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:37:10,894] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ google_api:0.8 /bin/bash init.sh ']
[2021-12-22 17:37:10,902] {subprocess.py:85} INFO - Output:
[2021-12-22 17:37:11,688] {subprocess.py:89} INFO - 0
[2021-12-22 17:37:11,688] {subprocess.py:89} INFO - 2
[2021-12-22 17:37:11,688] {subprocess.py:89} INFO - 3
[2021-12-22 17:37:11,688] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2264.jpg
[2021-12-22 17:37:12,772] {subprocess.py:89} INFO - [Info] /home/myamya/project/image_files/images/2264.jpg: Find face from in position [31, 39, 621, 725] and skew angle [0.83626616, 1.2407098, -0.2966359]
[2021-12-22 17:37:13,076] {subprocess.py:89} INFO - [Info]  /home/myamya/project/image_files/images/2264.jpg: Crop face [31, 39, 621, 725] and write it to file : /home/myamya/project/image_files/cropped_image/2264.jpg
[2021-12-22 17:37:13,076] {subprocess.py:89} INFO - training 1
[2021-12-22 17:37:13,076] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2266.jpg
[2021-12-22 17:37:13,738] {subprocess.py:89} INFO - [Info] /home/myamya/project/image_files/images/2266.jpg: Find face from in position [31, 39, 621, 725] and skew angle [0.83626616, 1.2407098, -0.2966359]
[2021-12-22 17:37:14,059] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2266.jpg: Duplicated image
[2021-12-22 17:37:14,059] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2263.jpeg
[2021-12-22 17:37:14,542] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2263.jpeg: cannot process file : 'y'
[2021-12-22 17:37:14,542] {subprocess.py:89} INFO - @@ /home/myamya/project/image_files/images/2265.jpeg
[2021-12-22 17:37:14,996] {subprocess.py:89} INFO - [Error] /home/myamya/project/image_files/images/2265.jpeg: cannot process file : 'y'
[2021-12-22 17:37:14,996] {subprocess.py:89} INFO - ## label 1 has 1 of training data
[2021-12-22 17:37:14,997] {subprocess.py:89} INFO - number of datas per label  [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
[2021-12-22 17:37:15,265] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:37:15,280] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=face_crop, execution_date=20211220T002700, start_date=20211222T083710, end_date=20211222T083715
[2021-12-22 17:37:15,305] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:37:15,321] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
